### DISCUSSION TITLE
Migrating to GraphQL

### DISCUSSION THREAD

**Discussion by LaansDole** [SchBenedikt/schbenedikt.github.io#22] [Ideas]
Hey @SchBenedikt, I just saw this video about GitHub GraphQL API, I think this can be an extra features for our project and might fix some potential bugs from Rest API 
https://www.youtube.com/watch?v=c_rJ3UfgVjg&list=PLm_Qt4aKpfKhlcKEhWcxevsrdNcWZuwyF&index=2

**Comment by SchBenedikt** [comment]
https://docs.github.com/de/graphql/overview/explorer
Yes, that could be a good idea.
